Cabal/Distribution/Simple/Configure.hs

```diff
@@ -42,6 +42,7 @@ module Distribution.Simple.Configure (configure,
                                       getInstalledPackages,
                                       getInstalledPackagesMonitorFiles,
                                       getPackageDBContents,
+                                      checkPackageDBs,
```

Comment:
Bikeshedding: I don't think there is any particular reason for this function to live in this module.